src/main/java/seedu/duke/command/EditNoteCommand.java

```diff
@@ -14,24 +24,109 @@
     public static final String COMMAND_USAGE = COMMAND_WORD + ": Edits a note in the notebook. Parameters: "
             + PREFIX_DELIMITER + PREFIX_INDEX + " INDEX "
             + "[" + PREFIX_DELIMITER + PREFIX_TITLE + " TITLE] "
-            + "[" + PREFIX_DELIMITER + PREFIX_LINE + " LINE]"
+            + "[" + PREFIX_DELIMITER + PREFIX_LINE + " INDEX STRING]"
```

Comment:
`INDEX STRING` is not very clear on what it does. Do you think we can use something like Line number or something?

Reply:
its not a line number. fixed it